### PR TITLE
Edit hero responsiveness

### PR DESCRIPTION
**Pull request by caocynthia**
Editing responsiveness again to make the spacing match the designs we got from the designers and make things scale upward better for larger screens

desktop max:
Screen Shot 2022-12-08 at 18 27 09

laptop:
Screen Shot 2022-12-08 at 18 30 07

mobile:
Screen Shot 2022-12-08 at 18 29 04

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -11,6 +11,10 @@
 }
 
 @layer base {
+  .mobile-heading {
+    @apply text-[40px] leading-[48px] text-white paytone;
+  }
+
```
Comment (srazh) [inline src/App.css]: can we remove extra white spaces?

File: src/components/about-us/AboutJosh.js [hunk 2]
```diff
@@ -1,26 +1,23 @@
...
 const AboutJosh = () => {
-    return (
-        <div className="bg-primary-50 py-10 ">
-
-            <div className="content flex py-2">
-                <img src={joshImage} className="h-[20rem] px-20 " />
-                <h1 class=" py-10 text-3xl text-dark font-bold drop-shadow-lg dmsans ">The Gruepr Backstory</h1>
-                <p class=" pl-[30rem] pr-[5rem] py-20 absolute text-gray-700 text-base dmsans dark">As an educator, I want my students to have the best possible experience during group work. I created gruepr because I was spending too much time assembling teams that accounted for my students' individual schedules and past experiences. I wanted a tool that could help me, and I wanted to keep it always free and open source for my fellow educators. The first version of the app was named "grouper," and that inspired my scuba-diving colleage to give me a fish logo. The name later changed to "gruepr" because the challenges of developing my first app reminded me of trying to avoid the fictional monster that haunts the dark in the first computer game I remember ever playing.
-            </p>
-
```
Comment (srazh) [inline src/components/about-us/AboutJosh.js:13]: same here! and reformat please

**Review by srazh — CHANGES_REQUESTED** [review]
overall lgtm! Please just run prettier + reformat and get rid of white spaces!

**Review by caocynthia — COMMENTED** [review]
fixed the whitespace! I think I changed things in the about section in another pull request so the code for those parts will probably be completely different later.

**Pull request merged**